package.json

```diff
@@ -99,7 +99,7 @@
     "@hot-loader/react-dom": "^16.13",
     "@reach/auto-id": "^0.13.0",
```

Comment:
- The text field character count looks like a button when compared to the rest.  Probably can just remove the black background to not make it look like a button.
- For the "Report Content" specifically, it is no longer right-aligned and becomes part of the label.

<img src="https://user-images.githubusercontent.com/64950861/150245294-661f1fc6-5b97-4ce8-bfd4-9f447b7c0a2d.png" width="300">
    <img src="https://user-images.githubusercontent.com/64950861/150244986-4b28502e-2c6a-4e68-806d-bc97473290d1.png" width="300">

Reply:
I basically agree with you, it shouldn't look like a button. However, removing the background looks kinda weird as well. I removed it for now, but I'll put it on my todo list and will experiment with some other solutions.

![image](https://user-images.githubusercontent.com/94839278/150324138-08e00a69-860b-43a1-ab31-dd36f6316b1a.png)

![image](https://user-images.githubusercontent.com/94839278/150324026-39e3940d-e14e-4859-a464-ba861a731d87.png)

Reply:
> _I'll put it on my todo list and will experiment with some other solutions._

:+1: